src/main/scala/io/treeverse/examples/List.scala

```diff
@@ -13,13 +10,17 @@ object List extends App {
       .scanLeft("")((a, b: String) => a + "/" + b)
 
   override def main(args: Array[String]) {
+    if (args.length != 2) {
+      Console.err.println("Usage: ... <repo_name> <commit_id>")
+      System.exit(1)
+    }
     val spark = SparkSession.builder().master("local").appName("I can list")
       .config("spark.hadoop.lakefs.api.url", "http://localhost:8000/api/v1/")
```

Comment:
Another thing to worry about: we pass credentials here on the session.  AFAICT this means a Spark application will not be able to access two lakeFS instances.  Doesn't need to be in this PR, but I hope Spark lets us do better and pass parameters  more easily to `newRDD`.